examples/js/loaders/BasisTextureLoader.js

```diff
@@ -427,6 +433,16 @@ THREE.BasisTextureLoader.BasisWorker = function () {
 
 		}
 
+		if ( !hasAlpha ) {
+			switch(config.format) {
+				case 9: // Hardcoded: THREE.BasisTextureLoader.BASIS_FORMAT.cTFPVRTC1_4_RGBA
+					config.format = 8; // Hardcoded: THREE.BasisTextureLoader.BASIS_FORMAT.cTFPVRTC1_4_RGB;
+					break;
+				default:
+					break;
+			}
+		}
```

Comment:
Could you try running:

```
npm install --global eslint @typescript-eslint/eslint-plugin @typescript-eslint/parser eslint-config-mdcs eslint-plugin-html
eslint --fix examples/js/loaders/BasisTextureLoader.js
```

That should fix the styleguide issues on the file. If that doesn't work, see https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2. We typically have empty lines after opening brackets and before closing brackets, and spaces after opening parens and before closing parens.

```js
if ( ! hasAlpha ) {

  switch ( config.format ) {

    // ...
```